server/services/store/sqlstore/legacy_blocks.go

```diff
@@ -0,0 +1,206 @@
+package sqlstore
+
+import (
+	"database/sql"
+	"encoding/json"
+
+	"github.com/mattermost/focalboard/server/utils"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/mattermost/focalboard/server/model"
+	"github.com/mattermost/focalboard/server/services/store"
+
+	"github.com/mattermost/mattermost-server/v6/shared/mlog"
+)
+
+// legacyBlocksFromRows is the old getBlock version that still uses
```

Comment:
I'm ok with this for now, but we should move this to code of the data migration and clean up the rest of the code.